src/Grpc/Interop/test/testassets/InteropWebsite/TestServiceImpl.cs

```diff
@@ -47,10 +47,9 @@ public override async Task StreamingOutputCall(StreamingOutputCallRequest reques
 
         foreach (var responseParam in request.ResponseParameters)
         {
-            // TODO(JamesNK): Remove nullable override after Grpc.Core.Api update
-            responseStream.WriteOptions = !(responseParam.Compressed?.Value ?? false)
+            responseStream.WriteOptions = (responseParam.Compressed?.Value ?? false)
```

Comment:
It looks like you flipped the logic here, there was a `!` (not) in front of the condition before